data/constants.js

```diff
@@ -39,6 +39,7 @@ const VAULT_CATEGORIES_IDS = {
   STABLE_POLYGON: 'STABLE_POLYGON',
   CONVEX: 'CONVEX',
   MESHSWAP: 'MESHSWAP',
+  AURA: 'AURA',
```

Comment:
Please also add to `VAULT_CATEGORIES_NAMES`, `AURA: 'Aura'`

Reply:
Fixed